src/js/components/SharedComponents/table/TablePaginatorItem.jsx

```diff
@@ -38,11 +40,11 @@ export default class SubmissionsTablePaginatorItem extends React.Component {
         let end = null;
 
         if (this.props.showLead) {
-            start = ' ... ';
+            start = <FontAwesomeIcon icon="ellipsis-h" className="pre" />;
```

Comment:
This is a confusing class name to me because of the html tag `<pre>`, which could be used as a css selector (`pre {...` vs `.pre {...`). Maybe change to `ellipsis-pre` and `ellipsis-post`?